tests/projects/external-kit/.dockerignore

```diff
@@ -0,0 +1,10 @@
+/.git
+/.gomodcache
+/build/*
+!/build/rpms/
+/build/rpms/*
+!/build/rpms/*.rpm
```

Comment:
You can delete this file because Twoliter is bringing its own .dockerignore now (so this one isn't being used).